.github/workflows/pr-open.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       digest: ${{ steps.builds.outputs.digest }}
     strategy:
       matrix:
-        package: [backend, frontend, migrations]
+        package: [backend, database, frontend, migrations]
```

Comment:
What is the matrix and what are the migrations here for?

Reply:
Migrations are from the template this project started with.  I'm pulling them in another PR right now!

Reply:
Matrix says to repeat this block swapping out package names each time.